stories/index.js

```diff
@@ -211,4 +211,10 @@ storiesOf("Marketing Thumb", module)
       text="hundreds of fishing experiences around the world handpicked by Fishtripr."
       />`
   );
+storiesOf("Mobile Payment Link", module)
+  .add("Basic", () =>
+    `<mobile-payment-link
+      link="https://www.imdb.com/title/tt0068646/?ref_=ttls_li_tt"
```

Comment:
I prefer to always use example.com links in non-production data. That domain is created to be used as a dummy placeholder. If you don't do this, you for example easily end up also using email addresses that can exist for real.